client/package.json

```diff
@@ -12,7 +12,8 @@
   "dependencies": {
     "next": "^14.2.13",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "zustand": "^5.0.0"
```

Comment:
Good work here, I like how you added zustand to the end because z is at the end of the alphabet!